### DISCUSSION TITLE
Is it possible to log to `context.log_file` with a different log level than used with stdout?

### DISCUSSION THREAD

**Discussion by gsingh93** [Gallopsled/pwntools#2109] [Q&A]
I want the logging from my code that's displayed on stdout to respect whatever `context.log_level` is at the time, but I'd like a log file generated that has all log messages, as if `context.log_level` was set to debug, but regardless of what the actual value of `context.log_level` is.

The motivation for this is that it would be nice to check a log file if an exploit fails rather than run the code again with a different log level or the `DEBUG` argument passed in.

**Comment by zachriggle** [comment]
I don't believe this was ever designed in.  The stack-based mechanism used to keep track of log levels in `context` is really annoying to duplicate, and using `log_file` at all is a niche thing.

The only thing I've used `log_file` for is headless exploit throwing during DEFCON or similar.
